src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-02-studio-70.md

```diff
@@ -0,0 +1,49 @@
+---
+title: October 2, 2017
+description: This version polished autocompletion in automation scripting, and improves flow between Modeler and Designer.
```

Comment:
_This version improves auto-completion in automation scripting and flow between Modeler and Designer._